src/index.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const fs = require('fs');
+const http = require('http');
+
+const server = new http.Server();
+
+const filename = './expences.json'
+
+server.on('request', (req, res) => {
+  if (req.url === '/info') {
+    const chunks = [];
+
+    req.on('data', (chunk) => {
+      chunks.push(chunk)
+    })
+
+    req.on('end', () => {
+      const rawData = Buffer.concat(chunks).toString()
+      const searchParams = new URLSearchParams(rawData);
+      const date = new Date();
+
+      const newEntry = {
+        title: searchParams.get('title'),
+        amount: searchParams.get('amount'),
+        date,
+      }
+
+      const file = fs.existsSync(filename)
+        ? fs.readFileSync(filename)
+        : '[]';
+
+      const expences = JSON.parse(file)
+
+      expences.push(newEntry);
+
+      const fileData = JSON.stringify(expences)
```

Comment:
This will format json to be human-readable
```suggestion
      const fileData = JSON.stringify(expences, null, 2);
```